### PR TITLE
test: add a test that produces an ArgumentOutOfBoundsException but shouldn't

### PR DESCRIPTION
**Pull request by ramezgerges**
GitHub Issue (If applicable): unoplatform/uno#12787

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by Youssef1313 — COMMENTED** [review]
I'm very surprised how the test case is just toooo small like that!

**Comment by jeromelaban** [comment]
The build is failing with `CSC : error CS1617: Invalid option '11' for /langversion. Use '/langversion:?' to list supported values.`, looks like we may need to adjust the version based on the targetframework.

**Comment by Youssef1313** [comment]
The CI installs .NET 6 which comes with compiler support to C# 10.

We can either use `<LangVersion>10</LangVersion>` or update CI to use .NET 7

**Review by Youssef1313 — COMMENTED** [review]
LGTM, as long as nothing breaks :)

File: src/Uno.XamlMerge.Tests/Scenarios/When_Different_Properties_Different_Namespaces/Input_Dictionary.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+		xmlns:skia="http://uno.ui/skia"
+		mc:Ignorable="skia">
+
+	<Style TargetType="ScrollViewer">
+		<Setter Property="Template">
+			<Setter.Value>
+				<ScrollBar
+						Height="16"
+						skia:Width="16" />
```
Comment (Youssef1313) [inline src/Uno.XamlMerge.Tests/Scenarios/When_Different_Properties_Different_Namespaces/Input_Dictionary.xml:13]: Can we have also another test case where `skia:Width` is before `Height`?

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
Looking good, thanks @ramezgerges!

**Pull request merged**